sdk/src/main/scala/com/horizen/storage/ErgoPlatformStorageAdapter.scala

```diff
@@ -0,0 +1,72 @@
+package com.horizen.storage
+import java.io.File
+import java.util
+import java.util.{Optional, List => JList}
+
+import com.horizen.utils._
+import javafx.util.{Pair => JPair}
+import org.ergoplatform.db.LDBFactory.factory
+import org.ergoplatform.db.VersionedLDBKVStore
+import org.iq80.leveldb.Options
+
+import scala.collection.JavaConverters._
+import scala.compat.java8.OptionConverters._
+
+
+/*
```

Comment:
LevelDB storage related interfaces are based on top of Array[Byte] instead of ByteArrayWrapper in IODB Store.
If we are going to remove IODB totally with Level DB implementation, we can think about reviewing of `Storage` interface:
1) Do we need to keep `ByteArrayWrapper` usage here?
2) review `Storage` methods in general.